src/containers/FormikForm/translateFormHooks.js

```diff
@@ -7,56 +7,27 @@
  */
 
 import { useState } from 'react';
+import dot from 'dot-object';
+import { merge } from 'lodash';
 import { fetchNnTranslation } from '../../modules/translate/translateApi';
 
-export function useTranslateForm(article, setArticle) {
+export function useTranslateApi(element, setElement, fields = {}) {
   const [translating, setTranslating] = useState(false);
 
-  const translateArticle = async () => {
+  const translateFunc = async () => {
```

Comment:
forslag:
```suggestion
  const translateFunction = async () => {
```
Evt noe som `translateToNn` siden nå kan det tenkes at den oversetter til alt mulig av språk, men den oversetter jo bare til nynorsk.